### PR TITLE
Fixed type #83

### PR DESCRIPTION
**Pull request by ramesht007**
## Description
Wrote detailed Inline documentations for tf_spider.py to attend to issue #83  in the Threepio project.

### CODE REVIEW

File: docs-crawler/docs/docs/spiders/tf_spider.py [hunk 1]
```diff
@@ -6,46 +12,73 @@
...
+    version = "2.1"  # Version of TesnorFlow documentation to crawl
+    allowed_domains = ['tensorflow.org']  # Crawls only links from TensorFlow
+
+    # Base URL for crawling
     start_urls = [
-        f'https://www.tensorflow.org/versions/r{version}/api_docs/python/tf'
-    ]
+        f'https://www.tensorflow.org/versions/r{version}/api_docs/python/tf']
+
+    # Regex rules for compiling a string to a Regex object.
```
Comment (Nolski) [inline docs-crawler/docs/docs/spiders/tf_spider.py:23]: The comment here should explain what the regex rules are. Here the rules will match on two groups and expects a function call similar to below
`foo.bar(arg1, arg2)`
First group is `foo.bar`.
Second group is `arg1, arg2`

File: docs-crawler/docs/docs/spiders/tf_spider.py [hunk 2]
```diff
@@ -6,46 +12,73 @@
...
         self.logger.info(f'Scraping {response.url}')
         item = ApiItem()
+
+        # Crawls the selector to create a list of each python doc.
         function_header = response.css('.lang-python')
         if len(function_header) == 0:
             return
+
+        # Preprocess the func_header and stores the processed representation.
+        # text = remove_tags(function_header.get())\
```
Comment (Nolski) [inline docs-crawler/docs/docs/spiders/tf_spider.py]: I think this line can be removed :)

**Review by Nolski — CHANGES_REQUESTED** [review]
This looks great, just a few things to fix and then it should be ready to merge. Great work!

File: docs-crawler/docs/docs/spiders/tf_spider.py (hunk 2 above)
Comment (ramesht007) [reply docs-crawler/docs/docs/spiders/tf_spider.py]: ok

**Review by Nolski — APPROVED** [review]

**Pull request merged**